.github/CODEOWNERS

```diff
@@ -685,11 +685,11 @@
 
 # Dynamic Feature Gating project
 /core/controllers/feature_flag*.py @oppia/web-android-compatibility-reviewers
```

Comment:
We only need web-android-compatibility-reviewers on the part of this that are config definitions, e.g. names of feature flags or things that have an Android-specific customization. The rest (all the plumbing) should be assigned to core-reviewers. Could you please update as needed?

Reply:
Updated.